test/unit/config.test.js

```diff
@@ -0,0 +1,102 @@
+const globals = require(`${__dirname}/../../app/globals`) // Always required
+const config = require(`${__dirname}/../../app/config`)
+
+describe('Config', () => {
+  it ('should export object', () => {
+    expect(config).toBeInstanceOf(Object)
+  })
+  describe('#getFrontendProps()', () => {
+    const input = {
+      apis: {
+        availableInFrontend: true,
+        doc: 'Connected APIs',
+        format: Array,
+        default: [],
+        publishId: {
+          availableInFrontend: true,
+          type: String,
+          default: ''
+        },
+        enabled: {
+          type: Boolean,
+          default: true
+        },
+        name: {
+          availableInFrontend: true,
+          format: String,
+          default: 'No Name'
+        },
+        host: {
+          availableInFrontend: true,
+          format: 'ipaddress',
+          default: '0.0.0.0'
+        },
+        port: {
+          availableInFrontend: true,
+          format: 'port',
+          default: 3000
+        },
+        database: {
+          availableInFrontend: true,
+          format: String,
+          default: ''
+        },
+        version: {
+          availableInFrontend: true,
+          format: String,
+          default: '1.0'
+        },
+        menu: {
+          availableInFrontend: true,
+          doc: 'Collection menu ordering and grouping',
+          format: Array,
+          default: []
+        },
+        credentials: {
+          format: Object,
+          clientId: {
+            format: String,
+            default: 'testClient'
+          },
+          secret: {
+            format: String,
+            default: 'superSecret'
+          }
+        }
+      }
+    }
+    const out = {
+      apis: {
+        publishId: true,
+        name: true,
+        host: true,
+        port: true,
+        database: true,
+        version: true,
+        menu: true
+      } 
+    }
+    it ('should return object', () => {
+      expect(config.getFrontendProps()).toBeInstanceOf(Object)
+    })
+    const schema = config.getFrontendProps(input, {})
```

Comment:
Test shouldn't create state that is used by other tests. If you need to reuse the result of an operation in multiple tests, you should either re-create that state in every single test or have a global variable that is updated on a `before`/`after` hook, that runs before of after each test.

A common pattern is to use a `before` to prepare the state for a test and an `after` to reset/cleanup, so that the next test to run has a blank canvas.